### PR TITLE
docs: add weekly report 0830-0912

### PR DESCRIPTION
**Pull request by SylviaBABY**
Changes:

add weekly report (0830-0912)

### CODE REVIEW

File: website/blog/2021-09-15-weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+title: "社区周报｜新晋 committer 两位，功能亮点更新进行中"
+keywords: 
+- Apache APISIX
+- API 网关
+- contributor
+- Good first issue
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [technology]
+---
+> 从 8.30 开始这两周有 33 位开发者为 Apache APISIX 提交了 130 个 commit。感谢以下小伙伴为 Apache APISIX 添砖加瓦（排名不分先后），是你们的无私付出，让 Apache APISIX 项目变得更好！
```
Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md]: commits

File: website/blog/2021-09-15-weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+
+我们还整理了一些适合新来社区的小伙伴们参加的 issue ！感兴趣的同学们，走过路过不要错过！
+
+## 贡献者统计
+
+从 8.30 开始这两周有 33 位开发者为 Apache APISIX 提交了 130 个 commit。感谢以下小伙伴为 Apache APISIX 添砖加瓦（排名不分先后），是你们的无私付出，让 Apache APISIX 项目变得更好！
```
Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md]: 8/30-9/12这两周

Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md]: commits

File: website/blog/2021-09-15-weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+apple
+apple
+apple
+```
+
+### Issue [#4945](https://github.com/apache/apisix/issues/4945)
+
+问题描述：根据相关指导文章，尝试在 M1 系统的 Macbook 上下载 Apache APISIX 2.9 版本时遇到问题。提示如下：
+
+```Apache
```
Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md:69]: "```shell"

File: website/blog/2021-09-15-weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  No file pcre.h in /include
+You may have to install PCRE in your system and/or pass PCRE_DIR or PCRE_INCDIR to the luarocks command.
+Example: luarocks install lrexlib-pcre PCRE_DIR=/usr/local
+make: *** [deps] Error 1
+```
+
+## 近期功能特性亮点
+
+- [referer-restriction 支持配置 blacklist 和 message](https://github.com/apache/apisix/pull/4916)(贡献者：okaybase)
+- [node_listen 和 admin_listen 支持更丰富的配置形式](https://github.com/apache/apisix/pull/4856)(贡献者：wayne-cheng),[补充参考](https://github.com/apache/apisix/pull/)4967
```
Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md]: ,[补充参考](https://github.com/apache/apisix/pull/)4967

`(https://github.com/apache/apisix/pull/)4967` to "(https://github.com/apache/apisix/pull/4967)"
"," to "，"

File: website/blog/2021-09-15-weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+- [admin-api 支持返回 stream 类型的插件信息](https://github.com/apache/apisix/pull/4947)(贡献者：spacewander)
+- [支持配置后退 SNI](https://github.com/apache/apisix/pull/5000)(贡献者：spacewander)
+- [proxy-mirror 支持按比例镜像请求](https://github.com/apache/apisix/pull/4965)(贡献者：okaybase)
+- [Control API 新增 dump 路由配置](https://github.com/apache/apisix/pull/5011)(贡献者：Zheaoli)
+- [dashboard支持Service Discovery 配置](https://github.com/apache/apisix-dashboard/pull/2081)(贡献者： bzp2010)
```
Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md]: "贡献者： " delete space after "："

File: website/blog/2021-09-15-weekly-report.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+- [dashboard Route 高级配置条件中支持内置参数配置](https://github.com/apache/apisix-dashboard/pull/1984)(贡献者： lookbrook)
```
Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md]: ditto

File: website/blog/2021-09-15-weekly-report.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+- [dashboard  Upstream 支持 Keepalive Pool 配置](https://github.com/apache/apisix-dashboard/pull/2117)(贡献者： bzp2010)
```
Comment (yzeng25) [inline website/blog/2021-09-15-weekly-report.md]: ditto

**Review by liuxiran — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**